.github/workflows/push-bundles.yaml

```diff
@@ -66,3 +69,33 @@ jobs:
         ENSURE_UNIQUE=1 \
         GITHUB_TOKEN=$(curl -s -X POST -H "Authorization: Bearer $(createJWT)" -H "Accept: application/vnd.github+json" "https://api.github.com/app/installations/${APP_INSTALL_ID}/access_tokens" | jq -r .token) \
```

Comment:
I don't think we need all this we no longer have `hack/pr-infra-deployments.sh` introduced in #240